File: phoenixfront/src/app/models/inventory/ReclamationDto.ts
import { AgentProdDto } from "./AgentProdDto";
import { ReclamType } from "./ReclamType";

export interface ReclamationDto {
    reclamationType?: ReclamType;
    reclamationText?:  string;
    senderReference?: string;
    receiverReference?: string;
    serialNumberOfReturnedStock?: string;
    campaignReference?: string;
    productType?: string;
    quantityToAdd?: number;
    reclamDate?: Date;
    vued?: boolean;
    agentProd?: AgentProdDto;

}

File: phoenix/keycloakuser/src/main/java/com/phoenix/dtostock/ReclamationDto.java
package com.phoenix.dtostock;

import lombok.*;

import java.time.LocalDate;

@Data
@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class ReclamationDto {
    private Long id;
    private ReclamType reclamationType;
    private String reclamationText;
    private String senderReference;
    private String receiverReference;
    private String serialNumberOfReturnedStock;
    private String campaignReference;
    private String productType;
    private int quantityToAdd;
    private LocalDate reclamDate;
    private Boolean vued;

}
